### PR TITLE
Allow users to abort before pushing to origin

### LINKED ISSUE #34: Add a confirmation step before pushing to users github repo
As a user
I'd like the ability to confirm the push to my github repo
so that I'm not accidentally surprised if I'm working on the wrong repo or have my origin set incorrectly.

The prompt should show where the push would go.
If the user chooses not to push changes, the command should terminate informing the user to perform the push.

### PR DESCRIPTION
**Pull request by IlleQuiProgrammat**
Fixes #34

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -405,6 +405,15 @@ log "Pushing initial cluster configuration"
 git add config.yaml footloose.yaml machines.yaml flux.yaml wks-controller.yaml
 
 git diff-index --quiet HEAD || git commit -m "Initial cluster configuration"
+
+# Confirm it works
+read -p "Would you like to push to origin? (y/n) " -n 1 -r
```
Comment (palemtnrider) [inline setup.sh]: Please print what the origin is including the branch.

File: setup.sh [hunk 2]
```diff
@@ -405,6 +405,15 @@ log "Pushing initial cluster configuration"
...
+echo
+# Checks if it is not a yes.. if so exit
+if [[ ! $REPLY =~ ^[Yy]$ ]]
+then
+    echo "To continue run `wksctl apply' against the repo and then `wksctl kubeconfig` after a push." && [[ "$0" = "$BASH_SOURCE" ]] && exit 1 || return 1
```
Comment (palemtnrider) [inline setup.sh]: The message needs to inform the user to perform a push before they can run a wksctl apply.

Comment (IlleQuiProgrammat) [reply setup.sh]: I see how this can be unclear.. noted and will be changed.

File: setup.sh (hunk 1 above)
Comment (IlleQuiProgrammat) [reply setup.sh]: Thank you, sorry for the late reply.. I believe the following commits fix this.

**Comment by IlleQuiProgrammat** [comment]
Apologies for not assigning myself and for the late reply.. I didn’t realise this would be an issue :/. I believe I have amended it appropriately.

File: setup.sh [hunk 3]
```diff
@@ -398,15 +398,27 @@ $sudo footloose status -o json > $status
 jk generate -f config.yaml -f $status setup.js
 rm -f $status
 
+GIT_ORIGIN = `git config --get remote.origin.url`
+GIT_BRANCH = `git rev-parse --abbrev-ref HEAD`
```
Comment (palemtnrider) [inline setup.sh]: These aren't valid in bash/zsh.  Can you elaborate on how you tested your changes?

Comment (IlleQuiProgrammat) [reply setup.sh]: They are, what isn’t valid about them?

**Pull request closed without merging**